flux/stdlib/influxdata/influxdb/to.go

```diff
@@ -0,0 +1,4 @@
+package influxdb
+
+// TODO(jsternberg): Implement the to method in influxdb 1.x.
```

Comment:
Is there an issue for this?

Reply:
I don't think so. It's been this way for awhile, but I think in previous syncs I was just deleting the file. I added this so I could remember that I did this.